doc/api/errors.md

```diff
@@ -2166,68 +2166,6 @@ added:
 An ESM loader hook returned without calling `next()` and without explicitly
 signaling a short circuit.
 
-<a id="ERR_MANIFEST_ASSERT_INTEGRITY"></a>
-
-### `ERR_MANIFEST_ASSERT_INTEGRITY`
```

Comment:
We have a section for documenting removed error codes: https://github.com/nodejs/node/blob/e0ed09d519d85970c33defc7d8a5b3c6892ecf5f/doc/api/errors.md#L3248-L3251